src/App.tsx

```diff
@@ -39,6 +40,7 @@ const App = ({ store }: Props) => (
             <DialogProvider>
               <div className={layout.stretch}>
                 <Routes />
+                <BetaCautionAlert />
```

Comment:
I mean, this works as well, but I'd like to have this just to set up the providers, and not display any actual components in it.

Can you please move it to the `pages` module, in one of the templates _(See: `RouteLayouts/Default`)_